index.js

```diff
@@ -0,0 +1,45 @@
+// const drbtn = document.querySelector(".Header__dropdown");
+// const drone = drbtn.getElementsByTagName("h5");
+// drone.style.border = '8px solid red';
+const test = document.querySelector(".Header");
+// test.style.border = "2px solid red";
+const drbtn = document.querySelector("h3");
+// drbtn.style.backgroundColor = 'green';
+const links = document.querySelector(".Header__links");
+links.style.backgroundColor = "white";
+links.style.display = "none";
+const btnBody = document.querySelector(".Header__dropdown")
+const lambda = document.querySelector(".Header__lambda");
+lambda.style.backgroundColor = "white";
+const google = document.querySelector(".Header__google");
+google.style.backgroundColor = "white";
+const mdn = document.querySelector(".Header__mdn");
+mdn.style.backgroundColor = "white";
```

Comment:
Still kinda confused as to why you'd want to style using javascript instead of CSS. I think you'd be better off leaving that for just CSS. separation of concerns and all that.